riscof-plugin/sail_cSim/README.md

```diff
@@ -0,0 +1,100 @@
+# Riscof Plugin for RISC-V SAIL C-Emulator
+
+This directory provides all the files and artifacts required for running SAIL RISC-V as a Reference model on the 
+[RISCOF](https://github.com/riscv/riscof) framework.
+
+## Getting SAIL RISC-V
+
+The model repository should be cloned from [here](https://github.com/rems-project/sail-riscv).
+
+## Building SAIL RISC-V C-Emulator
+
+The [README.md](../../README.md) at the top level of the sail-riscv repository gives details on 
+building C based executable of the model. You can build both the RV32 and RV64 variants.
+Once built, please add `$SAIL_RISCV/c_emulator` and `$SAIL_RISCV/ocaml_emulator` to your $PATH, 
+where $SAIL_RISCV is the path to the cloned repository.
```

Comment:
None of this makes sense, this is in sail-riscv

Reply:
Yes, it is in sail-riscv. Is it inaccurate?
The audience for this is people who are running this just to be able to use riscof, and the assumption is that this is where they will start. That assumption won't always be correct, but what matters is that sometimes it will be.

Reply:
The first section about cloning is completely unnecessary; they are already in the sail-riscv repo, either by browsing to it on GitHub or browsing to a local already-cloned copy.

The second section, fine, give a one-sentence pointer to the README.md at the top level, but "You can build both the RV32 and RV64 variants" seems completely unnecessary (it'd be highly surprising if that _weren't_ the case for the model).

Reply:
We may just build exactly what we want into a docker image that takes sail as is with YAML_config + test suite in, and  reference signatures out, which would move everything you find problematic out of riscv-sail repo entirely, so the PR may disappear